beacon-chain/attestation/service.go

```diff
@@ -163,7 +166,9 @@ func (a *Service) handleAttestation(ctx context.Context, msg proto.Message) erro
 // have been noted in the attestation pool. If not, it updates the
 // attestation pool with attester's public key to attestation.
 func (a *Service) UpdateLatestAttestation(ctx context.Context, attestation *pb.Attestation) error {
+	var err error
```

Comment:
Why is this needed?

Reply:
Still wondering why :)

Reply:
it's not needed, I forgot to push the latest commit : )